<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>吃掉蔡徐坤</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../usr/lib/eatcxk/chicken.png</normaloff>../../../../usr/lib/eatcxk/chicken.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>130</y>
      <width>0</width>
      <height>0</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>16</width>
       <height>16</height>
      </rect>
     </property>
     <widget class="QLabel" name="c">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>71</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>0</y>
        <width>71</width>
        <height>351</height>
       </rect>
      </property>
      <property name="text">
       <string>|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>0</y>
        <width>71</width>
        <height>351</height>
       </rect>
      </property>
      <property name="text">
       <string>|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>0</y>
        <width>71</width>
        <height>351</height>
       </rect>
      </property>
      <property name="text">
       <string>|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
|
</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="start_pushButton">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>189</y>
      <width>121</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>开始</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>61</y>
      <width>121</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Total:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>61</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>71</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Score:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>51</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSlider" name="slider">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>500</y>
      <width>591</width>
      <height>31</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="maximum">
     <number>581</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="stop">
    <property name="geometry">
     <rect>
      <x>336</x>
      <y>529</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>停止</string>
    </property>
   </widget>
   <widget class="QSlider" name="horizontalSlider">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>631</width>
      <height>20</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="sliderPosition">
     <number>1000</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="close">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>0</y>
      <width>31</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
